_data/api-swagger.yaml

```diff
@@ -649,6 +650,51 @@ definitions:
         }
       }
       ```
+
+  itsme Identification:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      properties:
+        PhoneNumber:
+          type: string
+          readOnly: true
+          description: |
+            The mobile phone number of the signer.
+            Must conform to E.164,
+            [the international public telecommunication numbering plan](https://en.wikipedia.org/wiki/E.164),
+            which requires the country calling code (Only the Belgian country calling code is supported: +32).
+          example: "+32123456789"
+        Attributes:
+          type: map of string
+          readOnly: true
+          description: |
+            Contains all available itsme Identification attributes.
+            These attributes may change, therefore we cannot guarantee the availability of any of these attributes.
+    example: |
+      An itsme Identification verification response may have the following content:
+      ```json
+      {
+          "Type": "itsme Identification",
```

Comment:
Should these names be in camel case? Which would also a require a change in the API PR.
@MrJoe